tests/unittest_brain.py

```diff
@@ -400,6 +401,12 @@ def test_nose_tools(self):
 
 @unittest.skipUnless(HAS_SIX, "These tests require the six library")
 class SixBrainTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        tox_env = os.environ.get("TOX_ENV_NAME")
+        if tox_env and "six" not in tox_env and HAS_SIX:
```

Comment:
`tox_env.endswith('-six')` instead of `"six" in tox_env` wouidn't be safer here?